paimon-core/src/main/java/org/apache/paimon/tag/Tag.java

```diff
@@ -201,4 +179,28 @@ public boolean equals(Object o) {
         return Objects.equals(tagCreateTime, that.tagCreateTime)
                 && Objects.equals(tagTimeRetained, that.tagTimeRetained);
     }
+
+    // =================== Utils for reading =========================
+
+    public static Tag fromJson(String json) {
+        return JsonSerdeUtil.fromJson(json, Tag.class);
+    }
+
+    public static Tag fromPath(FileIO fileIO, Path path) {
+        try {
+            return tryFromPath(fileIO, path);
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException(e.getMessage(), e);
+        }
+    }
+
+    public static Tag tryFromPath(FileIO fileIO, Path path) throws FileNotFoundException {
```

Comment:
Why not cache Tag and Changelog like Snapshot？

Reply:
Tag is mutable, it is danger.